todo/src/actions/index.js

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Common convention here is to save `const` variable for your action's names as strings. This makes it easy to import having "one source of truth" for what things are named. For example:
```
export const ADD_TODO = 'ADD_TODO';
export const SET_VISIBILITY_FILTER = 'SET_VISIBILITY_FILTER';
export const TOGGLE_TODO = 'TOGGLE_TODO';
```